go.mod

```diff
@@ -331,6 +331,8 @@ require (
 
 require github.com/libp2p/go-reuseport v0.1.0
 
+require github.com/gwos/tcg v0.0.0-20211123085413-fa76511f6546
```

Comment:
Any reason why it can't be in the big require block? And is it possible to use a tagged version? The repo seems to have a lot of tags..